dotCMS/src/main/webapp/html/ng-contentlet-selector.jsp

```diff
@@ -181,7 +182,12 @@
 
         function getCurrentUrlLanguageId () {
             var obj = window.location.href;
+
             const filter = "language_id=";
+
+            if (obj.indexOf(filter) === -1) {
+                return getSelectedLanguageId();
+            }
```

Comment:
Why we need this?

Reply:
because it searching for the language_id in the url and sometimes the url doesn't not have the language

Reply:
Can we do something that we ALWAYS have `language_id`?

Reply:
Done!